homeassistant/components/geniushub/climate.py

```diff
@@ -17,33 +17,29 @@
 
 GH_ZONES = ['radiator']
 
-GH_SUPPORT_FLAGS = \
-    SUPPORT_TARGET_TEMPERATURE | \
-    SUPPORT_ON_OFF
-
-GH_MAX_TEMP = 28.0
-GH_MIN_TEMP = 4.0
+# temperature is repeated here, as it gives access to high-precision temps
+GH_STATE_ATTRS = ['mode', 'temperature', 'type', 'occupied', 'override']
 
-# Genius Hub Zones support only Off, Override/Boost, Footprint & Timer modes
-HA_OPMODE_TO_GH = {
-    HVAC_MODE_OFF: 'off',
-    HVAC_MODE_AUTO: 'timer',
-    STATE_ECO: 'footprint',
-    STATE_MANUAL: 'override',
-}
-GH_STATE_TO_HA = {
-    'off': HVAC_MODE_OFF,
-    'timer': HVAC_MODE_AUTO,
-    'footprint': STATE_ECO,
+# GeniusHub Zones support: Off, Timer, Override/Boost, Footprint & Linked modes
+GH_MODE_TO_HA_PRESET = {
+    'off': None,
+    'timer': None,
+    'footprint': PRESET_ACTIVITY,
     'away': None,
-    'override': STATE_MANUAL,
-    'early': HVAC_MODE_HEAT,
+    'override': PRESET_BOOST,
+    'early': None,
```

Comment:
I don't think you need have None values inside this dict because you can use `GH_MODE_TO_HA_PRESET.get(...)` and this also return `None`

Reply:
Yeah, left over from some regression, maybe - fixed.